src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -51,6 +51,11 @@ export type ExpandableBlockProps = {
    * @default 'default'
    */
   size?: 'default' | 'small';
+  /**
+   * Whether or not the block is borderless.
+   * @default false
+   */
+  isBorderless?: boolean;
```

Comment:
Not sure about this prop, vs something like `styleType: 'default' | 'borderless'`. Any particular reason you went with a boolean?

Reply:
I wasn't sure about which one to go with, but I now remember you telling me use `styleType` for consistency. I'll change this

Reply:
This has been changed to styleType